vme/src/namelist.cpp

```diff
@@ -417,7 +415,7 @@ const int cNamelist::IsNameRawIdx(const char *name)
 
 const char *cNamelist::IsName(const char *name)
 {
-    char buf[MAX_STRING_LENGTH];
+    static char buf[MAX_STRING_LENGTH];
```

Comment:
Optimization found this bug, its return point to local destroyed variable on stack - quick fix make it static

Reply:
*applause*